src/utils/elementToSvg.ts

```diff
@@ -0,0 +1,35 @@
+import type { SatoriOptions } from "satori";
+import satori from "satori/wasm";
+
+import { loadGoogleFont } from "./loadGoogleFont.js";
+import { type HonoElement, toReactNode } from "./toReactNode.js";
+
+export type ElementToSvgOptions = Partial<SatoriOptions> & {
+  width?: number;
+  height?: number;
+};
+
+export async function elementToSvg(
+  element: HonoElement,
+  options: ElementToSvgOptions = {},
+) {
+  const { fonts, height = 630, width = 1200 } = options;
+
+  const svg = await satori(toReactNode(element), {
+    ...options,
+    width,
+    height,
+    fonts: fonts
+      ? fonts
+      : [
+          {
+            name: "Open Sans",
+            data: await loadGoogleFont({ family: "Open Sans", weight: 600 }),
+            weight: 500,
```

Comment:
This is needed for emojis and multi-language support: https://github.com/vercel/satori/blob/main/playground/pages/index.tsx#L561